brave/browser-laptop
# Fix styling of bookmarksToolbar.js

Also:
- Remove bookmarksToolbar.less (it is no longer necessary after #11530; See https://github.com/brave/browser-laptop/commit/405b0a4c4dda2be8e98f26e9f198f557ef9ce6de#diff-474538b5e876b82157a7e1b6228743e8)
- Specify the overflow indicator size

Closes #12066

Auditors: @cezaraugusto

Test Plan:
1. Add bookmarks as many as you see the overflow indicator on the bookmarks toolbar
2. Open about:preferences
3. Switch 'Bookmarks Bar' settings
4. Change the browser window's width
5. Make sure you see the overflow indicator on every setting

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
